src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
Performance (especially that kind of) of the queue plugin is irrelevant ;)
but yeah, for the sake of "clean defaults", why not.
In that case we need a DB migration, though.